futures-util/src/macros.rs

```diff
@@ -9,11 +9,11 @@
 /// # Example
 ///
 /// ```rust
-/// # use futures_util::pin_mut;
+/// # use core::pin::pin;
 /// # use core::pin::Pin;
 /// # struct Foo {}
 /// let foo = Foo { /* ... */ };
-/// pin_mut!(foo);
+/// let foo = pin!(foo);
```

Comment:
Please do not change this example because this is an example for pin_mut macro itself.

Reply:
Yes .. I should retain this the first time but seems some global editions touch this ><

Reverted.